modules/i18n/input-method/fcitx5.nix

```diff
@@ -37,6 +37,45 @@ in
           See [Using Fcitx 5 on Wayland](https://fcitx-im.org/wiki/Using_Fcitx_5_on_Wayland).
         '';
       };
+
+      config = lib.mkOption {
+        type = with lib.types; either path lines;
+        default = "";
+        description = ''
+          Configuration to be written to {file}`$XDG_DATA_HOME/fcitx5/conf/classicui.conf`
+        '';
+      };
```

Comment:
This feels a little weird for a generic `config` name to be mapped only to a `classicui.conf`.. but, I'm not familiar with `fcitx5`.